seed/api/v2_1/views.py

```diff
@@ -251,6 +253,36 @@ def hpxml(self, request, pk):
             xml = hpxml.export(property_view.state)
             return HttpResponse(xml, content_type='application/xml')
 
+    def _merge_relationships(self, old_state, new_state):
+        """
+        Merge the relationships between the old state and the new state. This is different than the version
+        in views/properties.py because if this is a buildingsync update, then the buildingsync file may
+        contain new or removed items that we want to merge.
+
+        :param old_state: PropertyState
+        :param new_state: PropertyState
+        :return: PropertyState, updated new_state
+        """
+        # for s in old_state.scenarios.all():
+        #     s.property_state = new_state
+        #     s.save()
+        #
+        # # Move the measures to the new state
+        # for m in PropertyMeasure.objects.filter(property_state=old_state):
+        #     m.property_state = new_state
+        #     m.save()
+        #
+        # # Move the old building file to the new state to perserve the history
+        # for b in old_state.building_files.all():
+        #     b.property_state = new_state
+        #     b.save()
+        #
+        # for s in Simulation.objects.filter(property_state=old_state):
+        #     s.property_state = new_state
+        #     s.save()
+
+        return new_state
```

Comment:
This code was intentionally commented out, right?

Reply:
yes